spec.bs

```diff
@@ -547,6 +547,7 @@ Add the following to the [=registry of standardized payment methods=] in
         DOMString payeeName;
         USVString payeeOrigin;
         AuthenticationExtensionsClientInputs extensions;
+        sequence<USVString> localeHint;
```

Comment:
Done for now, though I do want to do a quick review of other specs and see what we tend to do. Worried that a developer might set this and then be very upset when it isn't applied (since its just a hint).

Reply:
I checked with folks internally, and at least from what we can tell there is no consistency across specs here. Some specs use `foo`, some `fooHint`, and some `preferredFoo`. So I'm ok with using locale here, and having the spec text explain its hint nature.